### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,53 @@
...
 [![Circle CI](https://circleci.com/gh/grafana/metrictank.svg?style=shield)](https://circleci.com/gh/grafana/metrictank)
 [![Go Report Card](https://goreportcard.com/badge/github.com/grafana/metrictank)](https://goreportcard.com/report/github.com/grafana/metrictank)
 [![GoDoc](https://godoc.org/github.com/grafana/metrictank?status.svg)](https://godoc.org/github.com/grafana/metrictank)
 
+## Introduction
 
-## introduction
-*Metrictank is a cassandra-backed, metrics2.0 based, multi-tenant timeseries database for Graphite and friends*
+Metrictank is a multi-tenant timeseries engine for Graphite and friends.
+It provides long term storage, high availabalitity, efficient storage, retrieval and processing for large scale environments.
```
Comment (shanson7) [inline README.md]: availability*

File: README.md [hunk 2]
```diff
@@ -1,51 +1,53 @@
...
-* Backlog replaying and query responses, should be faster. [A Go GC issue may occasionally inflate response times](https://github.com/golang/go/issues/14812).
-* we use metrics2.0 in native input protocol and indexes, but [barely do anything with it yet](https://github.com/grafana/metrictank/blob/master/docs/tags.md).
+* No performance/availability isolation between tenants per instance. (only data isolation)
+* Minimum computation locality: we move the data from storage to processing code, which is both metrictank and graphite.
+* Backlog replaying and queries can be made faster. [A Go GC issue may occasionally inflate response times](https://github.com/golang/go/issues/14812).
+* We use metrics2.0 in native input protocol and indexes, but [barely do anything with it yet](https://github.com/grafana/metrictank/blob/master/docs/tags.md).
 * can't overwrite old data. We support reordering the most recent time window but that's it. (unless you restart MT)
 
-## interesting design characteristics (feature or limitation.. up to you)
+## Interesting design characteristics (feature or limitation.. up to you)
```
Comment (shanson7) [inline README.md]: ellipses should have 3 dots.

File: README.md (hunk 1 above)
Comment (Dieterbe) [reply README.md]: wow i really butchered that word.

Comment (DanCech) [reply README.md]: That's pretty impressessivive

**Comment by Dieterbe** [comment]
@woodsaj any feedback before merging?

**Pull request merged**